.circleci/config.yml

```diff
@@ -301,6 +402,18 @@ workflows:
           filters:
             branches:
               only: /^dependabot/submodules/.*/
+      # Publishing to Bintray is time consuming; let's do it manually for now
+      # while we figure out a way to add intelligence in the pipeline to only
+      # trigger it when there are changes that require it.
+      - Unblock Android React Native Bride Publishing to Bintray:
+          type: approval
+          requires: [ "Android Native Unit Tests" ]
+      - android-build-react-native-bridge:
+          name: Build Android RN Bridge JS Bundle
+          requires: [ "Unblock Android React Native Bride Publishing to Bintray" ]
```

Comment:
I tried to use the key for the job instead of the name, but it didn't work.

![Italian hand](https://media.giphy.com/media/A1bZQUmtLzOHm/source.gif)